packages/oui-header-tabs/src/header-tabs-divider.directive.js

```diff
@@ -0,0 +1,11 @@
+import template from "./header-tabs-divider.html";
+
+export default () => {
+    "ngInject";
+
+    return {
+        restrict: "E",
+        template,
+        replace: true
```

Comment:
Sadly, it's deprecated. A previous decision was made to not use it. 
We should use another way like `.replaceWith()` for example: https://docs.angularjs.org/api/ng/function/angular.element